pipeline-modules/app-service/gcp/go-scan.yaml

```diff
@@ -0,0 +1,56 @@
+provisioner: gcp
+name: go-scan
+version: 1
+revision: 1
+displayName: Go Scan
+description: Scans go code for vulnarabilities using gosec and golint
+target: ""
+keywords:
+  - Go
+  - Scan
+  - lint
+author: CloudCover
+meta: {}
+inputs:
+  properties:
+    go_sec_args:
+      title: Go sec args
+      description: Arguments for Go sec command
+      type: array
+      items:
+        type: string
+    go_lint_args:
+      title: Go lint args
+      description: Arguments for Go lint command
+      type: array
+      items:
+        type: string
+    working_dir:
+      title: Working directory
+      description: The current working directory to execute command
+      type: string
+      default: repo
+  required:
+    - go_sec_args
```

Comment:
why are these required? Isn't it possible to run these tools without any args?

Reply:
Updated ! removed required and added default commands to make it work without template inputs. 
But to keep in mind the default behavior of these commands is to break the pipeline if there is an issue discovered by the template. extra args are needed to let the build pass without failing eg. `-no-fail`